vars/sonarExecuteScan.groovy

```diff
@@ -40,7 +42,7 @@ void call(Map parameters = [:]) {
             // & `legacyPRHandling` & `inferBranchName`
             // writePipelineEnv needs to be called here as owner and repository may come from the pipeline environment
             writePipelineEnv(script: script, piperGoPath: piperGoPath)
-            Map stepConfig = readJSON(text: sh(returnStdout: true, script: "${piperGoPath} getConfig --stepMetadata '.pipeline/tmp/${METADATA_FILE}'${customDefaultConfig}${customConfigArg}"))
+            Map stepConfig = readJSON(text: sh(returnStdout: true, script: "${piperGoPath} getConfig --stepMetadata ${q('.pipeline/tmp/' + METADATA_FILE)}'${customDefaultConfig}${customConfigArg}"))
```

Comment:
```suggestion
            Map stepConfig = readJSON(text: sh(returnStdout: true, script: "${piperGoPath} getConfig --stepMetadata ${q('.pipeline/tmp/ + METADATA_FILE)}'${customDefaultConfig}${customConfigArg}"))
```

Reply:
There is an issue here,  thanks,  but your suggestion would not compile.